src/react-components/ui-root.js

```diff
@@ -1338,7 +1360,11 @@ class UIRoot extends Component {
                     ])}
                     value={this.state.pendingMessage}
                     rows={textRows}
-                    onFocus={e => e.target.select()}
+                    onFocus={e => {
+                      this.setState({ messageEntryOnTop: isMobile });
+                      e.target.select();
+                    }}
+                    onBlur={() => this.setState({ messageEntryOnTop: false })}
```

Comment:
This is buggy for me in Chrome for Android, the text box loses focus immediately after you tap on it and it moves to the top.

Reply:
Firefox for Android also gets into a bad state sometimes, where it doesn't exit/re-enter fullscreen mode correctly:

![image](https://user-images.githubusercontent.com/79419/53057802-a53b9f00-3465-11e9-901b-ad20550f1142.png)

Reply:
ah I forgot to test chrome on android, thanks for checking!

Reply:
weird couldn't get my local working on android. going to push to smoke and hope for the best, i think i figured this out :)